### PR TITLE
Depend on pronto 0.10.0

### LINKED ISSUE #6: stylelint 0.9.1 depends on older version of pronto
Similar to https://github.com/kevinjalbert/pronto-stylelint/issues/3, seems like an opportunity to bump the dependency of pronto? 🙂

Got this error on CI today:

```
pronto run -c origin/master -f github_pr_review github_status

Unable to activate pronto-stylelint-0.9.1, 
because pronto-0.10.0 conflicts with pronto (~> 0.9.0) (Gem::ConflictError)
```

### PR DESCRIPTION
**Pull request by zhuravel**
Fixes #6.

### CODE REVIEW

**Comment by kevinjalbert** [comment]
Awesome, I've been meaning to take a look at this.

Looks like the tests are failing, if you wanted to take a look at those. I can probably devote some time later this week though.

**Comment by zhuravel** [comment]
@kevinjalbert Fixed, tests are passing.

File: .travis.yml [hunk 1]
```diff
@@ -8,11 +8,10 @@ before_script:
 - npm install -g stylelint
 
 rvm:
-- 2.0
-- 2.1
-- 2.2
```
Comment (kevinjalbert) [inline .travis.yml:13]: These older versions aren't supported by pronto 0.10.x?

Comment (zhuravel) [reply .travis.yml:13]: Yes. https://github.com/prontolabs/pronto/commit/d1451fae34e41f0834a2da0a33ccd238fdb48e9b

**Comment by kevinjalbert** [comment]
@zhuravel Just released the new gem version -- https://rubygems.org/gems/pronto-stylelint/versions/0.10.0

Thanks again :)

**Pull request merged**